lib/mixins/disable-upgrade-mixin.js

```diff
@@ -15,6 +15,8 @@ import { dedupingMixin } from '../utils/mixin.js';
 
 const DISABLED_ATTR = 'disable-upgrade';
 
+let observedAttributesGetter;
```

Comment:
This is a lot of duplicated code; would be ideal to see if just using the mixin in `GenerateClassFromInfo` affects perf, and if not factor it so it can be used.